src/themes/device.js

```diff
@@ -20,6 +20,7 @@ const device = {
     mobileS: `(max-width: ${size.mobileS}px)`,
     mobileM: `(max-width: ${size.mobileM}px)`,
     mobileL: `(max-width: ${size.mobileL}px)`,
+    tabletM: `(min-width: ${size.mobileL}px) and (max-width: ${size.tabletL}px)`,
```

Comment:
Do we really need this? Or if it is just a special case, then you can use the hard breakpoints instead.

Reply:
Hi @sean-binary per this comment https://github.com/binary-com/deriv-com/pull/2495#discussion_r783680266 I gave , If I use that workaround provided there then this part won’t be needed and can remove it , just waiting to see if it's okay to make that change 🙏

Reply:
Yep please do it in that component level and remove it here 🙏  But as @suthesan-deriv was suggesting, can we not do it with `tabletL`/`tablet`/`tabletS` props?